packages/quantic/force-app/main/default/lwc/searchBoxStyle/searchBoxStyle.css

```diff
@@ -3,22 +3,46 @@
 .searchbox__container {
   display: flex;
   position: relative;
-  border-color: #e5e8e8;
   min-height: 48px;
   box-sizing: border-box;
   border-radius: 0.25rem;
-  overflow: hidden;
+  overflow: visible;
+  border: var(--lwc-borderWidthThin, 1px) solid var(--lwc-brandAccessible, #0176d3);
```

Comment:
Why are we adding a border to the container? The borders should already be in the inputs etc.